Gems/Atom/RPI/Code/Source/RPI.Public/RPISystem.cpp

```diff
@@ -250,22 +250,15 @@ namespace AZ
             AssetInitBus::Broadcast(&AssetInitBus::Events::PostLoadInit);
 
             // Update tick time info
-            FillTickTimeInfo();
+            m_tickTime.m_gameDeltaTime = deltaTime;
+            m_tickTime.m_currentGameTime = simulationTimeMs;
```

Comment:
suggestion: Should `m_gameDeltaTime` and `m_currentGameTime` also mention the units (s and Ms)? This seems like it could easily be a source of confusion or bugs